cmd/init.go

```diff
@@ -70,7 +70,9 @@ to quickly create a Cobra application.`,
 		trackers[pkg.GetDNSInfo] = &pkg.ActionTracker{Tracker: pkg.CreateTracker(pkg.GetDNSInfo, 1)}
 		trackers[pkg.TestHostedZoneLiveness] = &pkg.ActionTracker{Tracker: pkg.CreateTracker(pkg.TestHostedZoneLiveness, 1)}
 		trackers[pkg.CloneAndDetokenizeGitOpsTemplate] = &pkg.ActionTracker{Tracker: pkg.CreateTracker(pkg.CloneAndDetokenizeGitOpsTemplate, 1)}
-		trackers[pkg.CloneAndDetokenizeMetaphorTemplate] = &pkg.ActionTracker{Tracker: pkg.CreateTracker(pkg.CloneAndDetokenizeMetaphorTemplate, 1)}
+		trackers[pkg.CloneAndDetokenizeMetaphorJsTemplate] = &pkg.ActionTracker{Tracker: pkg.CreateTracker(pkg.CloneAndDetokenizeMetaphorJsTemplate, 1)}
+		trackers[pkg.CloneAndDetokenizeMetaphorGoTemplate] = &pkg.ActionTracker{Tracker: pkg.CreateTracker(pkg.CloneAndDetokenizeMetaphorGoTemplate, 1)}
+		trackers[pkg.CloneAndDetokenizeMetaphorFrontendTemplate] = &pkg.ActionTracker{Tracker: pkg.CreateTracker(pkg.CloneAndDetokenizeMetaphorFrontendTemplate, 1)}
```

Comment:
Do we need a tracker for each application, why not a single "metaphor"tracker with 3 or 4 steps.. it would make more sense.

Reply:
I believe it's nicer to know what exactly is happening, if it fails, it will be eaiser for the user to identify the issue